gate-web/src/main/groovy/com/netflix/spinnaker/gate/health/DownstreamServicesHealthIndicator.groovy

```diff
@@ -103,11 +105,13 @@ class DownstreamServicesHealthIndicator extends AbstractHealthIndicator {
         AuthenticatedRequest.allowAnonymous { service.health() }
       } catch (RetrofitError e) {
         serviceHealths[name] = "${e.message} (url: ${e.url})".toString()
+        log.error('Exception received during health check of service: {}, {}', name, serviceHealths[name])
```

Comment:
Logging only the highlevel exception message since this check runs every 30 seconds.